File: NXNavigationExtensionDemo/UIKit/Feature/Basic/Controllers/ViewController05_ShadowColor.m
//
//  ViewController05_ShadowColor.m
//  NXNavigationExtensionDemo
//
//  Created by Leo Lee on 2020/10/26.
//

#import <NXNavigationExtension/NXNavigationExtension.h>

#import "ViewController05_ShadowColor.h"

@interface ViewController05_ShadowColor ()

@end

@implementation ViewController05_ShadowColor

- (void)viewDidLoad {
    [super viewDidLoad];
    
}

- (UIColor *)nx_shadowImageTintColor {
    return [UIColor redColor];
}

- (UIColor *)nx_navigationBarBackgroundColor {
    return self.view.backgroundColor;
}

@end

File: Examples/Shared/UIKit/Basic/ViewController05_ShadowColor.swift
//
//  ViewController05_ShadowColor.swift
//  NXNavigationExtensionDemo
//
//  Created by lidan on 2021/11/8.
//

import UIKit

class ViewController05_ShadowColor: BaseViewController {

}

extension ViewController05_ShadowColor {
    
    override var nx_shadowImageTintColor: UIColor? {
        return .red
    }
    
    override var nx_navigationBarBackgroundColor: UIColor? {
        return view.backgroundColor
    }
    
}
